src/pages/mesh/basic/create-mesh.md

```diff
@@ -291,6 +291,20 @@ aio api-mesh:update update-mesh.json
     }
 ```
 
+## Delete a mesh
+
+To delete a mesh, run the following command:
+
+```bash
+aio api-mesh:delete
+```
+
+Refer to the [Command reference](../advanced/index.md#aio-api-meshdelete) for more information.
+
+<InlineAlert variant="info" slots="text"/>
+
+When you delete a mesh, the mesh's [caching configuration](../advanced/caching/index.md) is also deleted. You can either let the cache expire based on its preconfigured Time to Live (TTL) or [purge the cache](../advanced/caching/index.md#purge-the-cache).
+
```

Comment:
Any particular reason why we are talking about a mesh delete in the mesh create section?

Reply:
@revanth0212 Yes, i think this is the appropriate place. all CRUD commands are in this topic.